### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-07-07

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/CategoryTheory/Limits/Shapes/Pullback/Assoc.lean [hunk 1]
```diff
@@ -161,7 +161,9 @@ theorem hasPullback_assoc_symm [HasPullback f₁ (g₃ ≫ f₂)] : HasPullback
 
 /- Porting note: these don't seem to be propagating change from
 -- variable [HasPullback (g₂ ≫ f₃) f₄] [HasPullback f₁ (g₃ ≫ f₂)] -/
-variable [HasPullback (g₂ ≫ f₃) f₄] [HasPullback f₁ ((pullback.fst : Z₂ ⟶ X₂) ≫ f₂)]
+variable
+  [HasPullback ((pullback.snd : (pullback f₁ f₂) ⟶ X₂) ≫ f₃) f₄]
```
Comment (eric-wieser) [inline Mathlib/CategoryTheory/Limits/Shapes/Pullback/Assoc.lean:165]: Where did this come from?

**Review by Ruben-VandeVelde — APPROVED** [review]
Okay, I guess.

maintainer merge

File: Mathlib/CategoryTheory/Limits/Shapes/Pullback/Assoc.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/CategoryTheory/Limits/Shapes/Pullback/Assoc.lean:165]: There was a recent change to variables, making its elaboration a bit stricter, and I'm guessing a side effect is that it won't use the local notations above. Didn't investigate further.

**Comment by fpvandoorn** [comment]
LGTM

bors merge

**Pull request closed without merging**